### ISSUE TITLE
Throws parse error for DASH with SCTE35

### ISSUE THREAD

**Issue by aldafu** [emarsden/dash-mpd-cli#5]
Could this promising looking tool support downloading manifests that define SCTE35?

When trying to download DASH with manifests such as: https://docs.aws.amazon.com/mediatailor/latest/ug/dash-ad-markers.html

I.e. with:
```xml
<scte35:Signal>
    <scte35:Binary>/DAhAAAAAAAAAP/wEAUAAAHAf+9/fgAg9YDAAAAAAAA25aoh</Binary>
</scte35:Signal>
```

Then I get:

```
Download error: parse error parsing DASH XML: Parsing(
    "Unexpected `Event::Start(\"scte35:Signal\")`",
)
```

There wouldn't be any handling needed, just ignoring those SCTE35 events would be fine, since they are not needed to download the audio/video sources.

**Comment by emarsden** [comment]
Thanks, this is indeed a problem with the underlying dash-mpd-rs library. 

Do you know of public manifests that use these scte35 tags? I need some test material for the library.

**Comment by emarsden** [comment]
This should be fixed in release v0.1.12. There is support for various SCTE-35 elements, and elements that are not defined in the dash-mpd library should now be ignored instead of raising an error when parsing the manifest.
